packages/tooltip/src/components/popover/Tippy.module.css

```
@@ -133,4 +133,39 @@
   .tippy-box[data-theme~="light"] > .tippy-svg-arrow {
     fill: #fff;
   }
+
+  /*Dark mode!*/
+  .tippy-box[data-theme~="dark"] {
+    color: #26323d;
```

Vi kommer antagligen inte använda den här färgen ändå men för sakens skull skulle jag kört en ljus textfärg på mörk bakgrund. Kanske så?

```suggestion
    color: var(--lhds-color-ui-50);
```